curriculum/structure/superblocks/back-end-development-and-apis-v9.json

```diff
@@ -5,62 +5,14 @@
       "comingSoon": true,
       "modules": [
         {
-          "dashedName": "nodejs-core-libraries",
+          "dashedName": "introduction-to-nodejs",
           "comingSoon": true,
-          "blocks": []
+          "blocks": [
+            "lecture-working-with-nodejs-and-event-driven-architecture"
+          ]
         },
         {
-          "dashedName": "node-package-manager",
-          "comingSoon": true,
-          "blocks": []
-        },
-        {
-          "dashedName": "http-and-the-web-standards-model",
-          "comingSoon": true,
-          "blocks": ["lecture-understanding-the-http-request-response-model"]
```

Comment:
this is an existing lecture block, I don't believe you should delete everything here